shared/src/main/scala/com/raquo/domtypes/generic/defs/tags/SvgTags.scala

```diff
@@ -0,0 +1,733 @@
+    SvgFEMorphology <: SvgElement,
+    SvgFEOffset <: SvgElement,
+    SvgFEPointLight <: SvgElement,
+    SvgFESpecularLighting <: SvgElement,
+    SvgFESpotLight <: SvgElement,
+    SvgFETile <: SvgElement,
+    SvgFETurbulence <: SvgElement,
+    SvgFilter <: SvgElement,
+    SvgG <: SvgElement,
+    SvgImage <: SvgElement,
+    SvgLine <: SvgElement,
+    SvgLinearGradient <: SvgElement,
+    SvgMarker <: SvgElement,
+    SvgMask <: SvgElement,
+    SvgMetadata <: SvgElement,
+    SvgPath <: SvgElement,
+    SvgPattern <: SvgElement,
+    SvgPolygon <: SvgElement,
+    SvgPolyline <: SvgElement,
+    SvgRadialGradient <: SvgElement,
+    SvgRectElement <: SvgElement,
+    Svg <: SvgElement,
+    SvgStop <: SvgElement,
+    SvgSVG <: SvgElement,
+    SvgSwitch <: SvgElement,
+    SvgSymbol <: SvgElement,
+    SvgText <: SvgElement,
+    SvgTextPath <: SvgElement,
+    SvgTSpan <: SvgElement,
+    SvgUse <: SvgElement,
+    SvgView <: SvgElement
+  ] {
+  this: TagBuilder[T, SvgElement] =>
+
+
+  /**
+   * The altGlyph element allows sophisticated selection of the glyphs used to
+   * render its child character data.
+   *
+   * MDN
+   */
+  lazy val altGlyph:T[SvgElement] = tag("altGlyph")
+  
+  /**
+   * The altGlyphDef element defines a substitution representation for glyphs.
+   *
+   * MDN
+   */
+  lazy val altGlyphDef:T[SvgElement] = tag("altGlyphDef")
+
+  /**
+   * The altGlyphItem element provides a set of candidates for glyph substitution
+   * by the altGlyph element.
+   *
+   * MDN
+   */
+  lazy val altGlyphItem:T[SvgElement] = tag("altGlyphItem")
+  
+  /**
+   * The animate element is put inside a shape element and defines how an
+   * attribute of an element changes over the animation
+   *
+   * MDN
+   */
+  lazy val animate:T[SvgElement] = tag("animate")
+  
+  /**
+   * The animateMotion element causes a referenced element to move along a
+   * motion path.
+   *
+   * MDN
+   */
+  lazy val animateMotion:T[SvgElement] = tag("animateMotion")
+  
+  /**
+   * The animateTransform element animates a transformation attribute on a target
+   * element, thereby allowing animations to control translation, scaling,
+   * rotation and/or skewing.
+   *
+   * MDN
+   */
+  lazy val animateTransform:T[SvgElement] = tag("animateTransform")
+  
+  /**
+   * The circle element is an SVG basic shape, used to create circles based on a
+   * center point and a radius.
+   *
+   * MDN
+   */
+  lazy val circle:T[SvgCircle] = tag("circle")
+  
+  /**
+   * The clipping path restricts the region to which paint can be applied.
+   * Conceptually, any parts of the drawing that lie outside of the region
+   * bounded by the currently active clipping path are not drawn.
+   *
+   * MDN
+   */
+  lazy val clipPath:T[SvgClipPath] = tag("clipPath")
+  
+  /**
+   * The element allows describing the color profile used for the image.
+   *
+   * MDN
+   */
+  lazy val `color-profile`:T[SvgElement] = tag("color-profile")
```

Comment:
I think let's expose this as `colorProfile` instead, for consistency with our camelCase naming style.

Same for other backticked vals